repositories/dbmodels/db_api_keys.go

```diff
@@ -2,25 +2,27 @@ package dbmodels
 
 import (
 	"github.com/checkmarble/marble-backend/models"
+	"github.com/checkmarble/marble-backend/utils"
 
 	"github.com/jackc/pgx/v5/pgtype"
 )
 
 type DBApiKey struct {
-	Id             string      `db:"id"`
-	OrganizationId string      `db:"org_id"`
-	Key            string      `db:"key"`
-	DeletedAt      pgtype.Time `db:"deleted_at"`
-	Role           int         `db:"role"`
+	Id             string             `db:"id"`
+	OrganizationId string             `db:"org_id"`
+	Key            string             `db:"key"`
+	Description    string             `db:"description"`
+	DeletedAt      pgtype.Timestamptz `db:"deleted_at"`
```

Comment:
please remind me of the difference with pgtype.Time ?

Reply:
TBH, when doing this, I just dopy-paste this from `User` code.

After doing some research, it looks like `pgtype.Time` do not take the TS into account :
```go

// Timestamp represents the PostgreSQL timestamp type.
type Timestamp struct {
	Time             time.Time // Time zone will be ignored when encoding to PostgreSQL.
	InfinityModifier InfinityModifier
	Valid            bool
}

// Timestamptz represents the PostgreSQL timestamptz type.
type Timestamptz struct {
	Time             time.Time
	InfinityModifier InfinityModifier
	Valid            bool
}
```

Reply:
I found inside the `Timestamp` code this function :
```go 
func discardTimeZone(t time.Time) time.Time {
	if t.Location() != time.UTC {
		return time.Date(t.Year(), t.Month(), t.Day(), t.Hour(), t.Minute(), t.Second(), t.Nanosecond(), time.UTC)
	}

	return t
}
```

Reply:
In the end, the rule of thumb I use would be :
> Look at the sql schema, if this is `TIMESTAMP WITH TIME ZONE` use `pgtype. Timestamptz` else `pgtype.Time`